app/src/main/java/com/example/development/sakaiclientandroid/fragments/AllCoursesFragment.java

```diff
@@ -142,7 +141,7 @@ public void createTreeView(ArrayList<ArrayList<Course>> coursesSorted)
             //make a term header item, and make a treenode using it
             TermHeaderViewHolder.TermHeaderItem termNodeItem = new TermHeaderViewHolder.TermHeaderItem(termString);
             TreeNode termNode = new TreeNode(termNodeItem).setViewHolder(new TermHeaderViewHolder(mContext));
-            termNode.setClickListener(new TreeViewItemClickListener(treeView, root));
+            //termNode.setClickListener(new TreeViewItemClickListener(treeView, root));
```

Comment:
If code is unnecessary just get rid of it imo. It'll cause confusion otherwise (especially since its redundant due to the call to `setDefaultNodeClickListener`.